Tasks counter

fanaGebre opened the pull request.

fanaGebre committed:
this is counter branch

lbraun commented on html/index.html:
```diff
@@ -28,7 +28,7 @@ <h1>AngularJS and Leaflet Exploration</h1>
 
           <div class="col-md-6">
-            <h2>TODO List</h2>
+             <h2>TODO list({{counter}} tasks)</h2>
```
It looks like you added a space before `<h2>` that doesn't belong there.

Should there also be a space after "list"? So that it says "TODO list (1 tasks)" instead of "TODO list(1 tasks)".

Can you make it so that it says "TODO list (1 task)" instead of "TODO list (1 tasks)"? But still keep "TODO list (2 tasks)"? A little challenge for you 😄

lbraun commented on js/index.js:
```diff
@@ -16,7 +16,7 @@ app.controller("IndexController", ["$scope", "$http", function($scope, $http) {
 
   $scope.markers = new Array();
-
+  $scope.counter = 0;
```
Is it not possible to define this dynamically, something like `$scope.counter = $scope.markers.length`? I'm worried that these could get out of sync, and when we load them from the mongo database, the count won't be initialized properly.

lbraun commented on js/index.js:
```diff
@@ -66,6 +66,7 @@ app.controller("IndexController", ["$scope", "$http", function($scope, $http) {
   $scope.remove = function(index) {
     if (index !== -1) {
       $scope.markers.splice(index, 1);
+        $scope.counter--;
```
Why is this indented differently than the line that comes before it?

lbraun reviewed (changes requested):
Good work Fana! I left some suggested changes. Let me know if you have any questions or comments. Thanks!

fanaGebre committed:
this is updated counter tasks label

lbraun commented on html/index.html:
```diff
@@ -28,8 +28,8 @@ <h1>AngularJS and Leaflet Exploration</h1>
           <div class="col-md-6">
-            <h2>TODO List</h2>
             <table class="table">
+              <h2>TODO list {{counter <= 1? counter + " task" :  counter + " tasks" }}</h2>
```
This will still say "0 task" when it should say "0 tasks", right?

fanaGebre committed:
this is updated counter(for 0) tasks label

lbraun committed:
Update formatting

- Remove header 2 from table
- Move counter into a badge in the first row of the table to seperate it
from the title
- Fix indentation
